fortnite_api/shop.py

```diff
@@ -248,6 +250,63 @@ def __init__(self, *, data: Dict[str, Any], http: HTTPClientT) -> None:
         self.display_type: Optional[str] = data.get('displayType')
 
 
+class ShopEntryColors(ReconstructAble[Dict[str, Any], HTTPClientT]):
+    """
+    .. attributetable:: fortnite_api.ShopEntryColors
+
+    Represents the colors of a shop entry. This class inherits from :class:`~fortnite_api.ReconstructAble`.
+
+    Attributes
+    ----------
+    color1: :class:`str`
+        The first color of background gradient.
+    color2: Optional[:class:`str`]
+        The second color of background gradient. If not present, the gradient only consists of two colors.
+    color3: :class:`str`
+        The third color of background gradient.
+    text_background_color: :class:`str`
+        The fade out overlaying gradient color on which the text is displayed.
+    """
+
+    __slots__: Tuple[str, ...] = ("color1", "color2", "color3", "text_background_color")
+
+    def __init__(self, *, data: Dict[str, Any], http: HTTPClientT) -> None:
+        super().__init__(data=data, http=http)
+
+        self.color1: str = data['color1']
+        self.color2: Optional[str] = data.get('color2')
+        self.color3: str = data['color3']
+        self.text_background_color: str = data['textBackgroundColor']
+
+
+class ShopEntryRenderImage(ReconstructAble[Dict[str, Any], HTTPClientT]):
+    """
+    .. attributetable:: fortnite_api.ShopEntryRenderImage
+
+    Represents a render image for a shop entry. A render image is an image
+    used to visually represent a cosmetic item in the shop. This class inherits
+    from :class:`~fortnite_api.ReconstructAble`.
+
+    Attributes
+    ----------
+    product_tag: :class:`fortnite_api.ProductTag`
+        The product tag of the render image.
+    file_name: :class:`str`
+        The file name of the render image.
+    image: :class:`fortnite_api.Asset`
```

Comment:
Provide an example here if you could. I know what this is a file name but personally have no idea what that could be - may be helpful to users like myself.

Reply:
You refer to the `file_name`? 'm not 100% sure, but it seems fileName refers to the name of the file within the game files of the render image.

So what example you thought about?

Reply:
Apologies, I wasn't clear enough - Yes I'm referring to the `file_name` attribute. An example filename from the API so the user is aware of what the string looks like may be helpful. For example, `file.png` could be an example of a filename

Reply:
`file_name` refers to the name within the game files, but not the `image`. E.g. `T-Featured-Pickaxes-SleepyTimePickaxe`. This info isn't useful for 99% of the users and might be only relevant for datamining related tasks.

Not sure if we require more docs on this one. If you got something specific in mind, fell free to suggest a change.

Reply:
Ah, I see - I think that should be clarified in the docs for sure. Maybe something like:
```suggestion
    file_name: :class:`str`
        The internal file name of the rendered image. Refers to the name within the game files and **not** the :attr`image`. An example of this is ``T-Featured-Pickaxes-SleepyTimePickaxe``.
    image: :class:`fortnite_api.Asset`
```